### PR TITLE
Task Endpoint

### PR DESCRIPTION
**Pull request by jeffdyer**

### CODE REVIEW

File: src/task.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const code = objectFromID(ids[1]);
+    console.log("taskFromID() code=" + JSON.stringify(code));
+    const task = {lang: lang, code: code};
+    return task;
+  } catch (err) {
+    return err;
+  }
+}
+
+function postTask(task) {
```
Comment (KevinDyer) [inline src/task.js:47]: I think get and post should be async functions to facilitate forward compatibility

File: src/task.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  let id;
```
Comment (KevinDyer) [inline src/task.js:48]: Consider changing to ids and always having the POST /task api return a list of task ids?

File: src/task.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  if (task instanceof Array) {
+    id = [];
+    task.forEach((task) => {
```
Comment (KevinDyer) [inline src/task.js:51]: Consider using ids = task.map(postSingleTask);


function postSingleTask(task) {
 ...
 return id;
}

File: src/task.js (hunk 1 above)
Comment (jeffdyer) [reply src/task.js]: Okay. Done.

File: src/task.js (hunk 2 above)
Comment (jeffdyer) [reply src/task.js:47]: Good idea

File: src/task.js (hunk 3 above)
Comment (jeffdyer) [reply src/task.js]: Another good idea

**Pull request merged**